source/mod_boinc.f90

```diff
@@ -137,7 +146,8 @@ subroutine boinc_post
 
   use checkpoint_restart
 
-  write(boinc_logBuffer,"(a)") "Final checkpoint after tracking"
+  write(boinc_logBuffer,"(a)") "Tracking completed. Final checkpoint."
+  write(boinc_logBuffer,"(a,f8.3,a)") "Progress: ",99.0," %"
```

Comment:
I guess that if no postprocessing is given, this message should report 100%, shouldn't it?

Reply:
This write is redundant. Will remove it.